ubuntu/scripts/cleanup.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -e
+#!/bin/bash -ex
```

Comment:
What's the reason for `x` here? Doesn't look related to `apt` being non-interactive. Same question to other `-e` -> `-ex` changes.